### PR TITLE
Add globalization + fix CI

### PR DESCRIPTION
**Pull request by bajtos**
@superkhau please review

Connect to strongloop/loopback#2530

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "debug": "^2.2.0",
     "ioredis": "^2.2.0",
     "loopback-connector": "^2.4.0",
-    "msgpack5": "^3.4.0"
+    "msgpack5": "^3.4.0",
+    "strong-globalize": "^2.6.21"
```
Comment (superkhau) [inline package.json]: I think you're installing from staging. Should be `^2.6.5` ATM -- `^2.6.6` is better and fixes the bug I had during demo but I'm not sure if they released it yet.

**Comment by superkhau** [comment]
You will need to `npm install -g strong-globalize` and then run `slt-globalize -e` to generate the intl dir and verify the files in `intl/zz/message.json` to make sure all valid strings are translated. You'll need to update `.gitignore`, etc. See https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/globalization.md.

**Comment by bajtos** [comment]
@superkhau fixed, PTAL again.

**Comment by bajtos** [comment]
To fix failures on CIS, I need https://github.com/strongloop/loopback-datasource-juggler/pull/1044. To fix failures on Travis, a new 2.x version of juggler (including PR 1044) is needed.

File: lib/kv-redis.js [hunk 2]
```diff
@@ -2,11 +2,15 @@
...
 var Connector = require('loopback-connector').Connector;
-var debug = require('debug')('loopback:connector:kv-redis');
 var createPacker = require('./packer');
+var debug = require('debug')('loopback:connector:kv-redis');
 var Redis = require('ioredis');
+var SG = require('strong-globalize');
 var util = require('util');
 
+SG.SetRootDir(__dirname + '/..');
+var g = SG();
```
Comment (superkhau) [inline lib/kv-redis.js]: Needs to be first import in the file (right under 'use strict')

File: lib/kv-redis.js [hunk 3]
```diff
@@ -2,11 +2,15 @@
 
 var assert = require('assert');
 var Connector = require('loopback-connector').Connector;
-var debug = require('debug')('loopback:connector:kv-redis');
 var createPacker = require('./packer');
+var debug = require('debug')('loopback:connector:kv-redis');
 var Redis = require('ioredis');
+var SG = require('strong-globalize');
```
Comment (superkhau) [inline lib/kv-redis.js]: This too.

File: lib/kv-redis.js [hunk 4]
```diff
@@ -2,11 +2,15 @@
```
Comment (superkhau) [inline lib/kv-redis.js]:
```
var SG = require('strong-globalize');
SG.SetRootDir(__dirname + '/..');
var g = SG();
```

File: package.json [hunk 5]
```diff
@@ -24,9 +24,11 @@
     "debug": "^2.2.0",
     "ioredis": "^2.2.0",
     "loopback-connector": "^2.4.0",
-    "msgpack5": "^3.4.0"
+    "msgpack5": "^3.4.0",
+    "strong-globalize": "^2.6.6"
   },
   "devDependencies": {
+    "bluebird": "^3.4.1",
```
Comment (superkhau) [inline package.json]: unrelated, pls remove.

**Comment by superkhau** [comment]
@bajtos Reviewed. Two minor comments, otherwise LGTM. I'm not 100% convinced SG needs to be at the top of the file (and mess up our import ordering) but thats what was advised by @setogit. @setogit Can you chime in here?

File: package.json (hunk 5 above)
Comment (bajtos) [inline package.json]: This is the part "fix CI".

KVAO test suite requires `bluebird` to be installed, the same way as it requires a specific version of `should`. See https://github.com/strongloop/loopback-datasource-juggler/pull/1044/files

File: lib/kv-redis.js (hunk 4 above)
Comment (bajtos) [inline lib/kv-redis.js]: > ```js
var SG = require('strong-globalize');
SG.SetRootDir(__dirname + '/..');
var g = SG();
```

Fixed in ceec2e4.

**Comment by bajtos** [comment]
@superkhau fixed and replied, PTAL again.

**Comment by superkhau** [comment]
LGTM

**Comment by rmg** [comment]
There's a problem with this PR. The tests passed on ci.strongloop.com because they silently didn't run any tests. The failure on `cis-jenkins` appears to be accurate :-(

**Pull request merged**